pom.xml

```diff
@@ -98,6 +98,27 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <!-- Require upper bound dependencies -->
```

Comment:
This particular changes fixes the issues when using the default configuration in the ci.jenkins.io:
- https://github.com/jenkinsci/google-compute-engine-plugin/blob/develop/Jenkinsfile#L1

Errors:

```
14:01:51  [WARNING] Rule 4: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
14:01:51  Failed while enforcing RequireUpperBoundDeps. The error(s) are [
14:01:51  Require upper bound dependencies error for org.slf4j:jcl-over-slf4j:1.7.25 paths to dependency are:
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.slf4j:jcl-over-slf4j:1.7.25
14:01:51  and
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.jenkins-ci.main:jenkins-core:2.222.3
14:01:51      +-org.slf4j:jcl-over-slf4j:1.7.25 (managed) <-- org.slf4j:jcl-over-slf4j:1.7.26
14:01:51  , 
14:01:51  Require upper bound dependencies error for org.slf4j:log4j-over-slf4j:1.7.25 paths to dependency are:
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.slf4j:log4j-over-slf4j:1.7.25
14:01:51  and
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.jenkins-ci.main:jenkins-core:2.222.3
14:01:51      +-org.slf4j:log4j-over-slf4j:1.7.25 (managed) <-- org.slf4j:log4j-over-slf4j:1.7.26
14:01:51  , 
14:01:51  Require upper bound dependencies error for org.slf4j:slf4j-jdk14:1.7.25 paths to dependency are:
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.slf4j:slf4j-jdk14:1.7.25
14:01:51  and
14:01:51  +-org.jenkins-ci.plugins:google-compute-engine:4.2.1-SNAPSHOT
14:01:51    +-org.jenkins-ci.main:jenkins-war:2.222.3
14:01:51      +-org.slf4j:slf4j-jdk14:1.7.25 (managed) <-- org.slf4j:slf4j-jdk14:1.7.26
14:01:51  ]
```